.buildkite/ci.mjs

```diff
@@ -664,12 +664,15 @@ async function main() {
   let changedFiles;
   if (!isFork()) {
     console.log("Checking changed files...");
-    const baseRef = getCommit();
+    const baseRef = lastBuild?.commit_id || getTargetBranch() || getMainBranch();
     console.log(" - Base Ref:", baseRef);
-    const headRef = lastBuild?.commit_id || getTargetBranch() || getMainBranch();
+    const headRef = getCommit();
     console.log(" - Head Ref:", headRef);
 
     changedFiles = await getChangedFiles(undefined, baseRef, headRef);
+    for (const f of changedFiles) {
```

Comment:
`changedFiles` can be undefined intentionally, also the files are already printed out a few lines down